### ISSUE TITLE
Update web page describing how to disable Windows 11 PrtSc 

### ISSUE THREAD

**Issue by Jefferydo** [greenshot/greenshot#499]
The PrtSc troubleshooting instructions here are missing the step of disabling PrtSc in the accessibility controls:

https://getgreenshot.org/faq/my-prntscreen-key-no-longer-works-with-greenshot/

From chatGPT ...

To disable the native screenshot function of Windows 11 that's currently assigned to the PrtSc key, follow these steps:

1. Press the Windows + I button to open the Windows Settings.
2. Click on the Accessibility category.
3. Select Keyboard from the right-side menu.
4. Navigate to the option labeled "Use the Print Screen button to open screen snipping" and toggle off this option.

It worked for me.

**Comment by mariusi7** [comment]
Thanks! FYI, had to restart Greenshot to make it work.

**Comment by FiniteLooper** [comment]
I've done this and restarted Greenshot. The <kbd>PrtScn</kbd> button still does not work for me

![image](https://github.com/greenshot/greenshot/assets/463685/3efa0ad8-db44-444b-acff-34a6e33732fa)

**Comment by ccoenen** [comment]
This would be a very welcome addition to the FAQ, that was the first place I tried to find this info.

**Comment by neooblaster** [comment]
Hello,

I tried the solution from my side and that did not work.
I retry when I see this issue and that works as well !

Thanks for the tip and +1 to update the help page.

Best Regards.

**Comment by FiniteLooper** [comment]
@neooblaster uninstalling and then reinstalling Greenshot did the trick for me, thanks!  I don't know why I didn't think to try that.

For anyone else here's my steps:
* uninstall greenshot
* go to the **Settings > Accessibility > Keyboard** and turn off the preference about using the <kbd>PrtScn</kbd> button
* Install Greenshot

I guess Greenshot must do some kind of hooking up in the installation process, and a Windows update must have clobbered that.

**Comment by Anetre** [comment]
- In Windows turn off open Snipping Tool using the `Prntscrn` button
- In Greenshot settings rebind hotkeys for capture region
  - `Ctrl` + `Alt` + `Prntscrn` then Ok to save
  - `Prntscrn` Ok to save

**Comment by karenworld** [comment]
> * In Windows turn off open Snipping Tool using the `Prntscrn` button
> 
>     * In Greenshot settings rebind hotkeys for capture region
>       
>       * `Ctrl` + `Alt` + `Prntscrn` then Ok to save
>       * `Prntscrn` Ok to save

I tried that but Firefox is no longer displaying the captured screen.  Ever since windows 10 updated to 11.

**Comment by karenworld** [comment]
> @neooblaster uninstalling and then reinstalling Greenshot did the trick for me, thanks! I don't know why I didn't think to try that.
> 
> For anyone else here's my steps:
> 
>     * uninstall greenshot
> 
>     * go to the **Settings > Accessibility > Keyboard** and turn off the preference about using the PrtScn button
> 
>     * Install Greenshot
> 
> 
> I guess Greenshot must do some kind of hooking up in the installation process, and a Windows update must have clobbered that.

the steps you have shown doesn't work with windows 11 either....
